SEAPODYM_Data_Test.py

```diff
@@ -0,0 +1,94 @@
+from parcels import Particle, JITParticle, AdvectionRK4
+from parcels.grid import Grid
+from argparse import ArgumentParser
+from py import path
+import numpy as np
+
+
+def updateUserVars(particle, grid, time, dt):
+    for var in particle.user_vars.keys():
+        setattr(particle, var, getattr(grid, var)[time, particle.lon, particle.lat])
```

Comment:
If you capitalise the name and add documentation we could move this over to `particle.py` as a helper kernel. It clearly needs to state that this makes an assumption about particle and grid variables names though. Once we have parameterisable kernels we could even let the user define the name via a name map, but that is best left for later.